deploy/jenkins-ci/Jenkinsfile

```diff
@@ -140,6 +140,18 @@ node('kiali-build') {
           }
         }
       }
+
+     stage('Post Msg Bus Notification') {
+       when { environment name: 'RELEASE_TYPE', value: 'minor'}
+         steps {
```

Comment:
I'm getting an error on the `when` when trying it. I'm using "scripted pipeline" syntax, rather than "declarative" (sorry about that). So, you'll have to use a regular `if`, like here:
https://github.com/kiali/kiali/blob/a99c9db17483180409030c3a0390504b394feb10/deploy/jenkins-ci/Jenkinsfile#L92

Also, most likely, you'll have to omit the `steps`. Just write the instructions like in the other stages.